src/map/npc.cpp

```diff
@@ -6029,7 +6029,9 @@ int npc_reload(void) {
 	//TODO: the following code is copy-pasted from do_init_npc(); clean it up
 	// Reloading npcs now
 	for (nsl = npc_src_files; nsl; nsl = nsl->next) {
+#ifdef DEBUG
```

Comment:
Above the loop and after the loop the same messages as everywhere should be added.
Maybe harmonize the wording as well!
Cant suggest that on Github Web, sorry!

Reply:
Also as the TODO suggest, feel free to refactor this into a sub function.

Reply:
This will conflict with #7598, I'll make the changes in that one after this is merged.